src/main/java/org/opentripplanner/routing/edgetype/PathwayEdge.java

```diff
@@ -23,6 +24,7 @@
     private String name = "pathway";
 
     private boolean wheelchairAccessible = true;
+    private FeedScopedId id;
```

Comment:
Move id up as the first field. We try to keep things readable, and it is a common de-facto standard that ids are defined before other fields in entities.  The `serialVersionUID` is also defined after the constructor, could you clean it up a bit and move it up as the first declared field. I know you did not do that.